src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -0,0 +1,83 @@
+package com.easypost.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.easypost.EasyPost;
+import com.easypost.exception.EasyPostException;
+
+public class CreditCard extends BaseCreditCard {
+    public enum Priority {
+        PRIMARY,
+        SECONDARY
+    }
+
+    /**
+     * Fund a credit card.
+     *
+     * @param amount amount to fund.
+     * @param primaryOrSecondary primary or secondary payment method.
+     * @return CreditCardFund object.
+     * @throws EasyPostException when the request fails.
+     */
+    public static CreditCardFund fund(int amount, Priority primaryOrSecondary) throws EasyPostException {
+        return fund(amount, primaryOrSecondary, null);
+    }
+
+    /**
+     * Fund a credit card.
+     *
+     * @param amount amount to fund.
+     * @param primaryOrSecondary primary or secondary payment method.
+     * @param apiKey API key to use in request (overrides default API key).
+     * @return CreditCardFund object.
+     * @throws EasyPostException when the request fails.
+     */
+    public static CreditCardFund fund(int amount, Priority primaryOrSecondary, String apiKey) throws EasyPostException {
```

Comment:
`amount` should be a string based on the docs. This will allow for decimals as well. Our API will handle it all behind the scenes.

Reply:
fixed.